TipCatDotNet.Api/Controllers/PaymentController.cs

```diff
@@ -0,0 +1,49 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Identity.Web.Resource;
+using TipCatDotNet.Api.Models.HospitalityFacilities;
+using TipCatDotNet.Api.Services.HospitalityFacilities;
+
+namespace TipCatDotNet.Api.Controllers
+{
+    [AllowAnonymous]
+    [Route("api/payments")]
+    [Produces("application/json")]
+    [RequiredScope(ScopeRequiredByApi)]
+    public class PaymentController : BaseController
+    {
+        public PaymentController(IPaymentService paymentService)
+        {
+            _paymentService = paymentService;
+        }
+
+
+        /// <summary>
+        /// Gets receiver info by member code.
```

Comment:
Also that's not only a receiver info, but an account info too, and something else, maybe